src/Classes/ListControl.lua

```diff
@@ -346,7 +369,14 @@ function ListClass:OnKeyDown(key, doubleClick)
 			end
 		end
 	elseif #self.list > 0 and not self.selDragActive then
-		if key == "UP" then
+		if key == "a" then
+			if self.allowMultiselect and IsKeyDown("CTRL") then
+				self:WipeSelections()
+				for selId = 1, #self.list do
+					t_insert(self.selections, selId)
+				end
```

Comment:
This could probably just be a `copyTable` function.

Reply:
![image](https://user-images.githubusercontent.com/4302241/158338989-069bed40-4dca-4828-806a-79017c716960.png)
Whilst i'm not fully sure, I think self.list contains tables or other classes.
self.selections is a list of numbers.

x